tests/unit_tests/gui/tools/test_case_tool.py

```diff
@@ -27,3 +27,30 @@ def test_case_tool_init_prior(qtbot):
         list(storage.getStateMap())
         == [RealizationStateEnum.STATE_INITIALIZED] * ert.getEnsembleSize()
     )
+
+
+@pytest.mark.usefixtures("copy_poly_case")
+def test_that_case_tool_can_copy_case_state(qtbot):
```

Comment:
1. I think this fairly heavy test for what it's supposed to do. Ie, we instantiate the whole ERT machinery to see if the button works.
2. The test doesn't necessarily test what we want. Only that the ensemble has switched from `UNDEFINED` to `INITIALIZED` which doesn't mean that any data has been copied correctly.

I have no hard opinions on this, so we can merge if we can't come up with a better test.

Reply:
1. Agreed, but it is not terribly slow (400 ms) on my machine
1. Can make it more explicit if you want (testing value equality), but that requires even more boiler plate

I am hesitant to do more `mock`ing, as I might mock away what this test is here to check regression against. Do you have suggestions for improvement?

Reply:
I updated the test, checking that the parameters were copied over.

Reply:
Updated again with simpler case that runs faster (130 ms on my machine)